portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,21 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts = 
```

Comment:
This array could be constructed once and reused, since it doesn't change with state. You could move it outside the method and reference it as needed.

For such things, use ALL_CAPS for the variable name (JS convention). So something like:

const FACTS = [....facts....];
function addRandomFact() {
    // Something with FACTS[i];
}

Reply:
Okay, sounds great. I'll change that, thanks for the recommendation.